virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -25,12 +25,8 @@ include::snippets/technology-preview.adoc[]
 * You have installed the OpenShift CLI (`oc`).
 * You have link:https://docs.openshift.com/pipelines/latest/install_config/installing-pipelines.html[installed {pipelines-shortname}].
 
-
-include::modules/virt-deploying-ssp.adoc[leveloffset=+1]
-
 include::modules/virt-supported-ssp-tasks.adoc[leveloffset=+1]
 
-
 [id="example-pipelines_virt-managing-vms-openshift-pipelines"]
 == Example pipelines
 
```

Comment:
cc @ksimon1 I think 
Windows BIOS installer pipeline will be removed in 4.16?

Reply:
yes, only EFI pipeline will be present. The BIOS + Customize pipelines will be available only in upstream

Reply:
@geetikakay  @ksimon1 
If we remove all pipelines except EFI, the introductory paragraphs of "[Example pipelines](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/virtualization/virtual-machines#example-pipelines_virt-managing-vms-openshift-pipelines)" become irrelevant, correct? And by the way, why are the pipelines called 'example' if there aren't any other pipelines?

Reply:
Maybe it would be better to rename it from example pipelines to windows EFI pipeline?
The pipelines are called examples, because the installation of windows can be configured in many ways via the autounattend.xml file. We can't capture all ways. Thus we call the pipelines Examples, because user can see, how they can do it and modify it for their own use cases.

Reply:
@ksimon1 
I've modified the section to include only one pipeline. Please take a look. 
But how should the sections on running pipelines be modified?
"Running the example pipelines using the web console" - will the UI menu be called 'Pipeline' instead of 'Pipelines'? Are there any other changes?
"Running the example pipelines using the CLI" - this section is only about Windows 10. How should it be modified?